packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -418,6 +414,26 @@ export class TextFieldBase extends React.Component<ITextFieldProps, ITextFieldSt
     return errorMessage || '';
   }
 
+  /**
+   * Renders error message based on the type of the message.
+   *
+   * - If error message is string, it will render using the built in styles.
+   * - If error message is an element, user has full control over how it's rendered.
+   */
+  private _renderErrorMessage(): JSX.Element | null {
+    const errorMessage = this._errorMessage;
+
+    return errorMessage ? (
+      typeof errorMessage === 'string' ? (
+        <p className={this._classNames.errorMessage}>
+          <span data-automation-id="error-message">{errorMessage}</span>
+        </p>
+      ) : (
+        <div data-automation-id="error-message">{errorMessage}</div>
```

Comment:
You should probably still apply `this._classNames.errorMessage`--since it's been applied to JSX error messages in the past, not doing so would likely be a breaking behavior change for some people. If you need to override any of the styles with your custom rendering, you can do it via `ITextFieldProps.styles.errorMessage`.